Open Judge System/Data/OJS.Data.Models/ProblemSubmissionTypeSkeleton.cs

```diff
@@ -17,5 +17,7 @@ public class ProblemSubmissionTypeSkeleton
         /// Gets or sets a predefined skeleton for the task and strategy
         /// </summary>
         public byte[] SolutionSkeleton { get; set; }
+
+        public int? TimeLimit { get; set; }

```

Comment:
Could you add documentation because it is in Table with the name ProblemSubmissionTypesSkeletons and it is weird to see TimeLimit here without any explanation?

/// <summary>
        /// Gets or sets a time limit for the task and strategy
/// </summary>

Reply:
It will be good to rename the model, as now it holds more data than just the skeleton. Maybe `ProblemSubmissionTypeExecutionDetails`